app/controllers/braintree/payment_method_nonces_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Braintree::PaymentMethodNoncesController < ApplicationController
+
+  def create
+    render text: Braintree::PaymentMethodNonce.create(params[:payment_method_id]).payment_method_nonce.nonce
+  end
```

Comment:
I guess a full `js feature test` will be time-consuming, but if you have time for a controller test for this `#create`, I guess it would still test `PaymentMethodNonce`, which I'm not too sure where it's coming from (I presume it's a table)